pkg/cnab/config-adapter/adapter.go

```diff
@@ -179,7 +181,7 @@ func (c *ManifestConverter) generateBundleParameters(defs *definition.Definition
 		if !param.Destination.IsEmpty() {
 			p.Destination = &bundle.Location{
 				EnvironmentVariable: param.Destination.EnvironmentVariable,
-				Path:                param.Destination.Path,
+				Path:                manifest.ResolvePath(param.Destination.Path),
```

Comment:
In a follow-up PR I want to use this function so that people can use relative paths not just for the state files, but for params and creds. That way we should see a lot less of hard-coding absolute paths like /cnab/app/mystuff, when they just use "mystuff" and assume that everything is relative to the bundle directory.